reactive/kubernetes_master.py

```diff
@@ -2575,3 +2576,77 @@ def add_systemd_iptables_patch():
 
     # enable and run the service
     service_resume(service_name)
+
+
+@when('leadership.is_leader',
+      'kubernetes-master.components.started',
+      'endpoint.prometheus.joined',
+      'certificates.ca.available')
+def register_prometheus_jobs():
+    prometheus = endpoint_from_flag('endpoint.prometheus.joined')
+    # tls = endpoint_from_flag('certificates.ca.available')
```

Comment:
remove?

Reply:
This was actually a placeholder for improving the auth story that I forgot to circle back around to.  I'm going to go ahead and do that now.